02/app.js

```diff
@@ -1,7 +1,25 @@
-
 /* rozwiązanie z pętlą for */
-const x = 4;
 
+let x;
+
+do {
+    x = parseInt(prompt('Podaj liczbę z przedziału 1 - 9!'));
+  } while (x < 1 || x > 9);
+
+for (let i = 1; i < 10; i++) {
+    console.log(`${x} x ${i} = ${x * i}`);
+}
 
+/* rozwiązanie z pętlą while  */
 
-/* rozwiązanie z pętlą while  */
\ No newline at end of file
+let base = parseInt(prompt("Podaj podstawę:"));
+let exponent = parseInt(prompt("Podaj wykładnik:"));
+let result = 1;
+let i = 0;
+let string ='';
+while (i < exponent) {
+    string += `${base} * `;
+    i++;
```

Comment:
Przydałby się jeszcze jakiś `if` bo chyba nie do końca jest to poprawne co mam w konsoli

![image](https://user-images.githubusercontent.com/55095257/229024717-212e4189-4f04-4d7f-93c7-95e768fa2fc0.png)